_commands/basics/uniq.md

```diff
@@ -2,13 +2,13 @@
 ---
 
 uniq
---
-TODO: Add documentation for this command by submitting a pull request.
-<!-- one line explanation would go here -->
+------
+
+`uniq` allows a user to report or filter repeated lines that are adjacent in a file. If no input or output files are given, uniqu will read and write from standard input or output respectfully. 
 
 <!-- minimal example -->
 ~~~ bash
-*short* example of command usage and output
+uniq [options] in.txt out.txt
 ~~~
 
```

Comment:
We're looking for these pull requests to add a good amount of documentation to the commands. Take a look at the other pull requests here, or, go to [c4cs.github.io/reference](https://c4cs.github.io/reference) and click on any of those commands to see how filled out the pages are.